goth/runner/proxy.py

```diff
@@ -7,7 +7,7 @@
 
 from mitmproxy import options
 import mitmproxy.utils.debug
-from mitmproxy.tools import _main, cmdline, dump
+from mitmproxy.tools import main, cmdline, dump
```

Comment:
not sure you have this issue but am getting  `ImportError`
```
./scripts/download_release.py -v yagna
Traceback (most recent call last):
  File "/Users/pradeep/Data/Github/goth/./scripts/download_release.py", line 7, in <module>
    from goth.runner.download import (
  File "/Users/pradeep/Data/Github/goth/goth/runner/__init__.py", line 34, in <module>
    from goth.runner.proxy import Proxy, run_proxy
  File "/Users/pradeep/Data/Github/goth/goth/runner/proxy.py", line 10, in <module>
    from mitmproxy.tools import _main, cmdline, dump
ImportError: cannot import name '_main' from 'mitmproxy.tools' (/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/mitmproxy/tools/__init__.py)
```

Reply:
I just tested this on your branch with `python3.9` on Ubuntu 20.04 and I'm not getting the import error you mentioned. I'd revert these changes as we're primarly targeting Ubuntu 18.04 and 20.04 and both seem to be working fine (the former is running on our GitHub builders).